Sources/ProjectDescription/Template/Template.swift

```diff
@@ -131,6 +140,14 @@ public extension Template.File {
     static func file(path: String, templatePath: Path) -> Template.File {
         Template.File(path: path, contents: .file(templatePath))
     }
+
+    /// - Parameters:
+    ///     - path: Path where will be copied the folder
+    ///     - sourcePath: Path of folder which will be copied
+    /// - Returns: `Template.File` that is `.file`
+    static func directory(path: String, sourcePath: Path) -> Template.File {
```

Comment:
the naming of `Template.File` does not really make sense anymore. Maybe `Template.Item`? I am open to other suggestions as well

Reply:
do you think is ok change also the signatures to `files` to `items` too??
eg:
```
        let template = Template(
            description: "",
            attributes: [
                .required("name"),
                .optional("aName", default: "defaultName"),
                .optional("bName", default: ""),
            ],
            items: [
                .string(path: "static.swift", contents: "content"),
                .file(path: "generated.swift", templatePath: "generate.swift"),
                .directory(path: "destinationFolder", sourcePath: "sourceFolder"),
            ]
        )
```

Reply:
yes, but I'd also keep the current init and mark it deprecated, so this does not end up a breaking change.

Reply:
Added new init and the tag for deprecation.